### PR TITLE
update win xp rules

### PR DESCRIPTION
**Pull request by m8ttyB**
Update the rule set for the winxp tests

Reference work - https://bugzilla.mozilla.org/show_bug.cgi?id=1290737

### CODE REVIEW

File: tests/e2e/tests/test_redirects.py [hunk 1]
```diff
@@ -142,14 +142,17 @@ def _verify_winxp_redirect_rules(self, base_url, product_alias, user_agent,):
         }
         response = self._head_request(base_url, user_agent=user_agent, params=param)
         parsed_url = urlparse(response.url)
-        if product_alias in ['stub', 'latest', '44.0', '43.0.1']:
+        print 'product_alias: ' + product_alias
```
Comment (jgmize) [inline tests/e2e/tests/test_redirects.py]: Please remove this print statement.

**Comment by jgmize** [comment]
This needs a rebase.

File: tests/e2e/tests/test_redirects.py (hunk 1 above)
Comment (m8ttyB) [inline tests/e2e/tests/test_redirects.py]: Well that's embarrassing. Good catch.

**Comment by m8ttyB** [comment]
@oremj r?

**Comment by stephendonner** [comment]
r+

LGTM: http://pastebin.com/KcJYDSLj

sdonners-MacBook-Pro:go-bouncer sdonner$ git checkout update_xp_sha1_tests
Branch update_xp_sha1_tests set up to track remote branch update_xp_sha1_tests from origin.
Switched to a new branch 'update_xp_sha1_tests'
sdonners-MacBook-Pro:go-bouncer sdonner$ tox -e e2e -- --base-url=http://bouncer-bouncer.stage.mozaws.net/
<snip>
========================= 459 passed in 50.37 seconds ==========================
___________________________________ summary ____________________________________
  e2e: commands succeeded

**Comment by oremj** [comment]
lgtm

**Pull request merged**